haystack/pipelines/ray.py

```diff
@@ -139,36 +139,36 @@ def load_from_yaml(  # type: ignore
 
         Here's a sample configuration:
 
-            ```yaml
-            |   version: '1.0.0'
-            |
-            |    components:    # define all the building-blocks for Pipeline
-            |    - name: MyReader       # custom-name for the component; helpful for visualization & debugging
-            |      type: FARMReader    # Haystack Class name for the component
-            |      params:
-            |        no_ans_boost: -10
-            |        model_name_or_path: deepset/roberta-base-squad2
-            |    - name: MyESRetriever
-            |      type: ElasticsearchRetriever
-            |      params:
-            |        document_store: MyDocumentStore    # params can reference other components defined in the YAML
-            |        custom_query: null
-            |    - name: MyDocumentStore
-            |      type: ElasticsearchDocumentStore
-            |      params:
-            |        index: haystack_test
-            |
-            |    pipelines:    # multiple Pipelines can be defined using the components from above
-            |    - name: my_query_pipeline    # a simple extractive-qa Pipeline
-            |      type: RayPipeline
-            |      nodes:
-            |      - name: MyESRetriever
-            |        inputs: [Query]
-            |        serve_deployment_kwargs:
-            |          num_replicas: 2    # number of replicas to create on the Ray cluster
-            |      - name: MyReader
-            |        inputs: [MyESRetriever]
-            ```
+           ```yaml
+           version: '1.0.0'
+
+            components:    # define all the building-blocks for Pipeline
+            - name: MyReader       # custom-name for the component; helpful for visualization & debugging
+              type: FARMReader    # Haystack Class name for the component
+              params:
+                no_ans_boost: -10
+                model_name_or_path: deepset/roberta-base-squad2
+            - name: MyESRetriever
+              type: ElasticsearchRetriever
+              params:
+                document_store: MyDocumentStore    # params can reference other components defined in the YAML
+                custom_query: null
+            - name: MyDocumentStore
+              type: ElasticsearchDocumentStore
+              params:
+                index: haystack_test
+
+            pipelines:    # multiple Pipelines can be defined using the components from above
+            - name: my_query_pipeline    # a simple extractive-qa Pipeline
+              type: RayPipeline
+              nodes:
+              - name: MyESRetriever
+                inputs: [Query]
+                serve_deployment_kwargs:
+                  num_replicas: 2    # number of replicas to create on the Ray cluster
+              - name: MyReader
+                inputs: [MyESRetriever]
+           ```
```

Comment:
```suggestion
           ```yaml
           version: '1.0.0'

            components:    # define all the building-blocks for Pipeline
            - name: MyReader       # custom-name for the component; helpful for visualization & debugging
              type: FARMReader    # Haystack Class name for the component
              params:
                no_ans_boost: -10
                model_name_or_path: deepset/roberta-base-squad2
            - name: MyRetriever
              type:  BM25Retriever
              params:
                document_store: MyDocumentStore    # params can reference other components defined in the YAML
                custom_query: null
            - name: MyDocumentStore
              type: ElasticsearchDocumentStore
              params:
                index: haystack_test

            pipelines:    # multiple Pipelines can be defined using the components from above
            - name: my_query_pipeline    # a simple extractive-qa Pipeline
              type: RayPipeline
              nodes:
              - name: MyRetriever
                inputs: [Query]
                serve_deployment_kwargs:
                  num_replicas: 2    # number of replicas to create on the Ray cluster
              - name: MyReader
                inputs: [MyRetriever]
           ```
```